src/index.js

```diff
@@ -51,24 +53,41 @@ window.addEventListener("keydown", (e) => {
 });
 
 // Deleting
-todoContainer.addEventListener("click", (e) => {
-  const itemSelected = e.target.closest(".fa-list-icon");
+todoContainer.addEventListener('click', (e) => {
+  const itemSelected = e.target.closest('.fa-list-icon');
 
   if (!itemSelected) return;
 
   itemSelected
-    .querySelector(".fa-solid")
-    .classList.remove("fa-ellipsis-vertical");
+    .querySelector('.fa-solid')
+    .classList.remove('fa-ellipsis-vertical');
 
-  itemSelected.querySelector(".fa-solid").classList.add("fa-trash-can");
+  itemSelected.querySelector('.fa-solid').classList.add('fa-trash-can');
 
-  const trashIcon = itemSelected.querySelector(".fa-trash-can");
+  const trashIcon = itemSelected.querySelector('.fa-trash-can');
 
-  trashIcon?.addEventListener("click", () => {
-    const parentEle = trashIcon.closest(".todo");
+  trashIcon?.addEventListener('click', () => {
+    const parentEle = trashIcon.closest('.todo');
     const { index } = parentEle.dataset;
 
     deleteItem(+index);
     createList(getToDos());
   });
-});
\ No newline at end of file
+});
+
+let indexesToRemove = [];
+
+todoContainer.addEventListener('click', (e) => {
+  const activeEl = e.target.closest('.checkbox');
+  if (!activeEl) return;
+  const parentEl = activeEl.closest('.todo');
+
+  indexesToRemove = checkoffItem(parentEl);
+});
+
+clearListBtn.addEventListener('click', () => {
+  indexesToRemove.forEach((i) => {
+    deleteItem(i);
+  });
```

Comment:
- [ ] As stated by the previous reviewer, kindly fix the issue for complete todo as when I click on the checkbox, it is marked as completed in UI but it is not updated in local storage. Also, consider when refreshing the page the todos populate with todos from local storage.
>You can take a look at the following image to understand what I mean
>![image](https://user-images.githubusercontent.com/53402600/206877065-d5005d6f-0106-40ae-9d37-12eba8e0206d.png)